src/components/AppServicesPageNotFound/AppServicesPageNotFound.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { PageSection, Button } from "@patternfly/react-core";
+import { useHistory } from "react-router-dom";
+import {
+  AppServicesEmptyState,
+  AppServicesEmptyStateVariant,
+} from "../../components";
+
+const AppServicesPageNotFound: React.FunctionComponent = () => {
+  const { t } = useTranslation();
+
+  function GoHomeBtn() {
+    const history = useHistory();
+    function handleClick() {
+      history.push("/");
```

Comment:
This is absolutely not a concern of this repo, it belongs to app-services-ui

Reply:
Agreed. Similar to ErrorBoundary it's also using kas-ui for running kas-ui standalone. It will not wrap in app-services-ui app.
We have routing in kas-ui as well for running standalone.

Reply:
kas-ui will have to inject its own callback. No biggie.